java/connector-node/risingwave-connector-service/src/main/java/com/risingwave/connector/deserializer/StreamChunkDeserializer.java

```diff
@@ -107,6 +107,24 @@ static ValueGetter[] buildValueGetter(TableSchema tableSchema) {
                                 return row.getString(index);
                             };
                     break;
+                case TIMESTAMP:
```

Comment:
For `TIMESTAMPTZ` we can use the same logic.

Reply:
But there's no zone infomation from the NativeDateTime, is it acceptable?

Reply:
But I'm not sure how does it represent both datetime and timezone info into a single i64. I will add timestamptz 
 support once I figure it out.